src/cdk/scrolling/virtual-scroll-viewport.html

```diff
@@ -0,0 +1,8 @@
+<div #contentWrapper class="cdk-virtual-scroll-content-wrapper"
+     [style.transform]="_renderedContentTransform">
+  <ng-content></ng-content>
+</div>
+<div class="cdk-virtual-scroll-spacer"
```

Comment:
Add comment like
```
Spacer used to force the scrolling container to the correct size for the
*total* number of items so that the scrollbar captures the entire data set.
```